include/OpenSpaceToolkit/Core/Containers/Array.hpp

```diff
@@ -53,7 +53,7 @@ class Array : public std::vector<T>
 
         /// @brief              Default constructor (disabled)
 
-                                Array                                       ( ) = delete ;
+                                Array                                       ( )  ;
```

Comment:
To review

Reply:
The automatic casting used in `ArrayCasting` seem to require a default constructor to compile and cast the `Array` object properly.